README.md

```diff
@@ -0,0 +1,149 @@
+
+### Notes:
+- **`-v` (verbose) output** will display detailed logs.
+- **`--k6_script`** should point to the K6 test script you're using.
+
+---
+## Test Script
+The main test script (`test-script.js`) simulates virtual users (VUs) performing HTTP GET requests. The script is designed with two load phases and uses K6's `ramping-vus` and `constant-vus` executors to control how VUs are scaled.
+
+Here’s an example of the script:
+```javascript
+import http from 'k6/http';
+import { check, sleep } from 'k6';
+
+// Configuration for virtual users, ramp-up, and test duration
+const target = __ENV.VUS || 300;
+const rampupTime = __ENV.RAMPUP || "5s";
+const duration = __ENV.DURATION || "1m";
+
+export const options = {
+  scenarios: {
+    rampUp: {
+      executor: 'ramping-vus',
+      startVUs: 0,
+      stages: [{ duration: rampupTime, target: target }],
+      tags: { rampUp: 'true' },
+    },
+    instantLoad: {
+      executor: 'constant-vus',
+      vus: target,
+      duration: duration,
+      startTime: rampupTime,
+      tags: { rampUp: 'false' },
+    },
+  },
+  thresholds: {
+    'http_req_failed{rampUp:false}': [{ threshold: 'rate==0', abortOnFail: true }],
+    'http_req_duration{rampUp:false}': [{ threshold: 'p(95)<1000', abortOnFail: true }],
+  },
+  summaryTrendStats: ['avg', 'min', 'med', 'max', 'p(90)', 'p(95)'],
+};
+
+// Main function executed by each virtual user
+export default function () {
+  http.get('http://localhost:3000/channel');
+  http.get('http://localhost:3000/channel/create');
+  sleep(1); // Simulate time between requests
+}
+```
+
+---
+## Authentication Setup (Optional)
+
+If your test requires JWT authentication, you can set up the login flow as follows:
+
+```javascript
+export function setup() {
+  const loginHeaders = { 'Content-Type': 'application/json' };
+
+  const loginResponse = http.post('http://localhost/auth/login', JSON.stringify({
+    name: 'your_username',
+    password: 'your_password',
+  }), { headers: loginHeaders });
+
+  const isLoginSuccessful = check(loginResponse, {
+    'login successful': (res) => res.status === 200 && res.json('accessToken') !== undefined,
+  });
+
+  if (!isLoginSuccessful) {
+    throw new Error('Login failed');
+  }
+
+  return loginResponse.json('accessToken');
+}
+
+export default function (accessToken) {
+  const authHeaders = { Authorization: `Bearer ${accessToken}` };
+
+  http.get('http://localhost:3000/channel', { headers: authHeaders });
+  http.get('http://localhost:3000/channel/create', { headers: authHeaders });
+  sleep(1);
+}
+```
+
+---
+
+## Thresholds and Validation
+The following performance thresholds are defined for validation:
+- **HTTP request failures**: The failure rate should be 0% (`rate==0`).
+- **HTTP request duration**: 95% of requests should complete within 1000ms (`p(95)<1000`).
+
+If the thresholds are exceeded, the test will be aborted.
+
+---
+## Endpoints Supported
```

Comment:
alle http methods mogen behalve `trace`, want trace is stom. Dus : post, get, options, put, delete, patch, connect en head.